framework/src/main/java/org/apache/felix/framework/Felix.java

```diff
@@ -367,7 +367,9 @@ public Felix(Map configMap)
         }
         else
         {
-            m_logger = new Logger();
+        	String type = (String) m_configMutableMap.get(FelixConstants.LOG_TYPE_PROP);
+        	LoggerType loggerType = "jul".equals(type) ? LoggerType.JUL : LoggerType.STDOUT; 
```

Comment:
Why not LoggerType.valueOf ? This equals check fails on extending the LoggerType enum